src/components/Toolbar.js

```diff
@@ -10,13 +54,62 @@ const shortcutIds = [
   'search',
 ]
 
-export const Toolbar = connect(({ settings: { dark } }) => ({ dark }))(({ dark }) =>
-  <div className="toolbar">
-    {shortcutIds.map(id => {
-      const { name, svg: Icon, exec } = shortcutById(id)
-      return <div key={id} className="toolbar-icon" title={name} onClick={exec}>
-        <Icon fill={dark ? 'white' : 'black'} />
-      </div>
-    })}
-  </div>
-)
+const ToolbarMessageOverlay = ({ classname = 'toolbar-overlay', name, description }) => {
+    return <TransitionGroup>
+            <CSSTransition key={0} timeout={200} classNames='fade'>
+                <div className={classname}>
+                    <div className={'overlay-name'}>{ name }</div>
+                    <div className={'overlay-body'}>{ description }</div>
+                </div>
+            </CSSTransition>
+        </TransitionGroup>
+}
+
+var holdTimer
```

Comment:
`var` is not permitted in this project. You can use `let` with `// eslint-disable-line fp/no-let` in this particular instance since I don't have `redux-thunk` integrated yet for cleaner handling of side effects and asynchronous operations.

Reply:
Ok